tests/test_oct.py

```diff
@@ -86,11 +86,12 @@ def test_image_builder_oct(
         )
 
     assert result.consumed_files == {dest}
-    assert len(result.new_images) == 2
+    assert len(result.new_images) == 1
     for result in result.new_images:
         expected_values = expected_oct_properties
         if "fundus" in result.name:
-            expected_values = expected_fundus_properties
+            # expected_values = expected_fundus_properties
+            continue  # Skip fundus_images for now
```

Comment:
Rather than using continue I think it's better to mark the test as xfail, but I don't know how that would fit in with parametrization.